configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
@pwolfram: since config.defaults contains *all* of our settings, I would add this line in it, but commenting it out for the time being (maybe with a documentation comment right above it to say what it is, how one would choose it, etc).